File: tests/expected/assert.nim
proc compare_assert(a: int, b: int) =
  assert(a == b)
  assert(not (0 == 1))

proc main() =
  assert(true)
  assert(not (false))
  compare_assert(1, 1)
  assert(true)
  assert(true)
  echo "OK"

main()

File: tests/expected/assert.dart
// @dart=3.4
import 'package:sprintf/sprintf.dart';

compare_assert(int a, int b) {
  assert(a == b);
  assert(!(0 == 1));
}

main(List<String> argv) {
  assert(true);
  assert(!(false));
  compare_assert(1, 1);
  assert(true);
  assert(true);
  print(sprintf("%s", ["OK"]));
}
